pkg/validation/test.go

```diff
@@ -665,10 +665,13 @@ func validateDependencies(fieldRoot string, dependencies []api.StepDependency) [
 	return errs
 }
 
-func validateDNSConfig(fieldRoot string, searches []api.StepDNSConfig) (ret []error) {
+func validateDNSConfig(fieldRoot string, dnsConfig []api.StepDNSConfig) (ret []error) {
 	var errs []error
-	for i, search := range searches {
-		if search.Searches[i] == "" {
+	for i, dnsconfig := range dnsConfig {
+		if dnsconfig.Nameservers[i] == "" {
+			errs = append(errs, fmt.Errorf("%s.nameservers[%d] must be set", fieldRoot, i))
```

Comment:
Didn't you say it is optional?

Reply:
The field itself is optional but we require it to provide entry for our coredns pod running on sshd-bastion for step-registry migration of the MA jobs.

Reply:
Yeah but right now if DNSConfig is set it must set both Nameservers and Searched. I would have thought that one of them can be enough, is that wrong?

Reply:
Searches is only going to work if we pass in the ip to the openshift cluster's nameserver so it can resolve the coredns-ppc64le.bastion or coredns-z.bastion.... address